database/game_data_importer.py

```diff
@@ -148,7 +148,7 @@ def parse_review_file(filepath):
 
 def main():
     # This is a cardinal sin when it comes to security, but who gives a damn its a local db!
-    with psycopg.connect(host='localhost', dbname='game_reviews', user='main', password='access123', port=5432) as conn:
+    with psycopg.connect(host='localhost', dbname='postgres', user='main', password='access123', port=5432) as conn:
```

Comment:
Honestly what I did here is bad practice but was just a quick and dirty solution to minimize setup. In reality we would want to set these parameters as environment variables which we can then import. to use the game_reviews database, you will need to run a create db command within the postgres client but for our purposes, it really is just silly semantics